README.md

```diff
@@ -25,105 +25,140 @@ Maven 3.6
 -	Go to the SDP UI and create a project workshop-samples. If you are running samples with SDP cluster.
 
 ## Configuring Standalone Pravega and running
--	Clone Pravega from https://github.com/pravega/pravega.git and get required version.
--	Get proper release version Ex: pravega-r0.8
+-	Clone Pravega ([getting-started-guide](https://github.com/pravega/pravega/blob/master/documentation/src/docs/getting-started/quick-start.md)) from https://github.com/pravega/pravega.git and get required version.
```

Comment:
You can also link to here: https://pravega.io/docs/latest/getting-started/quick-start/